src/vuestic-theme/vuestic-components/vuestic-checkbox/VuesticCheckbox.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+    <div class="form-check abc-checkbox abc-checkbox-primary" :class="{'abc-checkbox-circle': circle}">
```

Comment:
there are multiple classes allowed (abc-checkbox-primary, abc-checkbox-danger etc just like in bootstrap)

It would be cool to add a property that could control the colors via these  predefined classes
1) default is primary
2) you can use v-class and/or computed prop to control the color
2) property must be among allowed list of values ['primary', 'danger', ... ] https://alligator.io/vuejs/property-validation/